Monika After Story/game/script-topics.rpy

```diff
@@ -13046,3 +13046,149 @@ label mas_show_unseen:
+            m 1eka "That's okay, [player]."
+            m 3eub "Until you know what you want to program, I can teach you how to use Python!
+            m 1rka "That is, if you have the same version of Python as I do...{w=0.5}"
+            m 1eub "What version of Python did you download?{nw}"
+            $ _history_list.pop()
+            menu:
+                m 1eub "What version of Python did you download?{fast}"
+                "option one":
+                    m 1hksdlb "...{w=0.5}"
+                    m 1rksdla "[player], I use the other version of Python...{w=0.5}"
+                    m 1lksdlb "Well, that's okay...{w=0.5}"
+                    m 1eka "I hope you have fun, anyway."
+                    m "...{w=0.5}"
+                    m "Hey, [player]?"
+                    m "Have I been underestimating your coding skills?{nw}"
+                    $ history_list.pop()
+                    menu:
+                        m "Ahaha, I'm sorry, [player]!"
+                        m "I didn't mean to- I just never thought to ask."
+                    "No":
+                        m "Okay, good."
+                        m "I just wanted to make sure I wasn't assuming your skill level."
+                        
+                "option two":
+                    m 1hub "Great!"
+                    m 1eub "That's the version of Python I use."
+                    m 3rksdlb "Also, [player]?"
+                    m "Have I been underestimating your coding skills?{nw}"
+                    $ _history_list.pop()
+                    menu:
+                        m "Have I been underestimating your coding skills?{fast}"
+                        "Yes":
+                            m "Ahaha, I'm sorry, [player]!"
+                            m "I didn't mean to- I just never thought to ask."
+                        "No":
+                            m "Okay, good."
+                            m "I just wanted to make sure I wasn't assuming your skill level.
+    
+        "To help a friend.":
+            m 3eub "That's great!"
+            m 1eka "You're so selfless, [player]."
+            m 1hub "I can't wait to see what you create!"
+            m 1eub "What version of Python did you download?{nw}"
+            $ _history_list.pop()
+            menu:
+                m 1eub "What version of Python did you download?{fast}"
+                "option one":
+                    m 1hksdlb "..."
+                    m 1rksdla "[player], I use the other version of Python...{w=0.5}"
+                    m 3rksdlb "So we can't really help each other that much...{w=0.5}"
+                    m 1hub "But that's okay!"
+                    m 1eub "I hope you have fun helping your friend."
+                "option two":
+                    m 1hub "Great!"
+                    m 1eub "That's the version of Python I use."
+                    m 1hua "Maybe both you and your friend could help me out once you know how to code!
```

Comment:
Same thing here regarding reusing menu.

Reply:
Would this be the same thing as I've seen in the code where it jumps to another event? And would I just create a label and then jump to that event?

Reply:
`jump` is different to `call`. You'll want to call the label (as calls will return back to the line below that statement when completing the label)

Reply:
Ahh I see. Thanks!

Reply:
I hope I did it correctly-